.eslintrc.cjs

```diff
@@ -0,0 +1,149 @@
+/** @type {import("@types/eslint").Linter.Config} */
```

Comment:
Quick links & descriptions for the plugins this config loads in:

* [`@typescript-eslint/eslint-plugin`](https://typescript-eslint.io): rules that validate TypeScript syntax and/or are more powerful because of TypeScript - configured with the [strictest typescript-eslint configs](https://typescript-eslint.io/linting/configs#strict-type-checked) 🤘
* [`eslint-plugin-eslint-comments`](https://mysticatea.github.io/eslint-plugin-eslint-comments): verifies good usage of ESLint directive comments (e.g. `// eslint-disable-line`)
* [`eslint-plugin-deprecation`](https://github.com/gund/eslint-plugin-deprecation): reports on usage of anything marked as `@deprecated` in a JSDoc tag
* [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc): validates properly formatted JSDoc comments (`/** ... */`) and their tags (`@param`, etc.)
* [`eslint-plugin-markdown`](https://github.com/eslint/eslint-plugin-markdown): lints embedded <code>```</code> code blocks in Markdown files [^1]
* [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n): Node-specific rules (a fork of `eslint-plugin-node` as that original package seems to be not maintained anymore)
* [`eslint-plugin-no-only-tests`](https://github.com/levibuzolic/eslint-plugin-no-only-tests): stops `.only` & similar from being checked in accidentally
* [`eslint-plugin-perfectionist`](https://eslint-plugin-perfectionist.azat.io): sorts imports, properties, etc. - with auto-fixers for all rules, thank goodness!
* [`eslint-plugin-regexp`](https://ota-meshi.github.io/eslint-plugin-regexp): helps find mishaps with regular expressions
* [`eslint-plugin-vitest`](https://github.com/veritem/eslint-plugin-vitest): Vitest-specific rules, akin to `eslint-plugin-jest`

[^1]: Not to be confused with [Markdownlint](https://github.com/DavidAnson/markdownlint), which is a separate linter specifically for Markdown files. I haven't investigated using `eslint-plugin-md(x)` yet (https://github.com/JoshuaKGoldberg/create-typescript-app/issues/567).